corekit/src/main/java/com/midtrans/sdk/corekit/core/MidtransSDK.java

```diff
@@ -114,7 +114,7 @@ private MidtransSDK(@NonNull BaseSdkBuilder sdkBuilder) {
 
         String deviceType = null;
         if (context != null) {
-            mPreferences = SdkUtil.newPreferences(context, LOCAL_DATA_PREFERENCES);
+            mPreferences = context.getSharedPreferences(LOCAL_DATA_PREFERENCES, Context.MODE_PRIVATE);
```

Comment:
I think you can remove this line of code, instead of changing it into sharedPreference

Reply:
I was thinking like that, but it will impact all data will be not stored in local, I'm worried it will impact to current flow not running well. so I'm decided to still use shared preference. too many dependencies, if we remove shared preference for now, I have the plan to release this first and see there is have a problem or no. If no we can continue to remove all local data handlers. except somebody knows the flow for each function that used local data. as far I know every transaction will save the user detail to local data first even if the transaction using Snap Token, it will save to local data first after that will query from local data on the next process.